src/twisted/internet/test/test_process.py

```diff
@@ -862,6 +862,8 @@ class PTYProcessTestsBuilder(ProcessTestsBuilderBase):
     if platform.isWindows():
         skip = "PTYs are not supported on Windows."
     elif platform.isMacOSX():
+        skip = "PTYs are flaky from a Darwin bug. See #8840."
```

Comment:
This might be a bit overly broad.  PTYs should totally be *supported* on OS X (and they do work, to the extent that the platform works), just there are a few tests which are inadvertently exercising edge-cases around `close()`.